README.md

```diff
@@ -1,3 +1,15 @@
+# Important: deprecation alert
+This library is being deprecated, please start migrating to [RN-UILib](https://github.com/wix/react-native-ui-lib/).  
```

Comment:
I think worth saying that the repo will not be maintained anymore, but the component has moved to uilib.. 
otherwise it sounds like we're removing the component entirely.

Reply:
Fixed.